desserts/chocolate chip oatmeal cookies.md

```diff
@@ -0,0 +1,40 @@
+# Chocolate chip oatmeal cookies
+
+These nutty cookies are inspired by Food Network Kitchen's [oatmeal-flax chocolate chip cookies](https://www.foodnetwork.com/recipes/food-network-kitchen/oatmeal-flax-chocolate-chip-cookies-recipe-2106797) and Trader Joe's gluten-free oatmeal cookie recipe (found on the back of their rolled oats.
+
+
+## Ingredients (for 12 cookies)
+
+- 2 tbsp butter
+- 3.5 tbsp white sugar
+- 3 tbsp brown sugar
+- 1 egg
+- 1/4 tsp baking soda
+- 1/4 tsp vanilla
+- 1 tbsp chopped walnuts
+- 1/2 tbsp flaxseeds
+- 3 tbsp flour
+- 1/8 cup peanut butter
+- pinch of salt
+- pinch of cinnamon
+- 1/4 cup chocolate chips
+- 1/2 cup rolled oats
+
+
+## Instructions
+
+1. Preheat oven to 350°F. Line a baking sheet with parchment paper.
+
+2. Soften, but don't melt, the butter (2 tbsp). With a mixer on medium-high, beat the butter with the white sugar (3.5 tbsp) and brown sugar (3 tbsp) until fluffy. Whisk in the egg.
```

Comment:
I think you're supposed to start at a slow speed to start mixing the butter and sugar so the sugar doesn't explode everywhere. Then slowly increase the speed to medium.